### ISSUE TITLE
Assignee images do not show

### ISSUE THREAD

**Issue by ZooeyMiller** [dwyl/tudo#231]
Currently the`src` of the image tags for the assignees is just the assignee's username, with no alt tags or actual image, we need to update the link so it actually shows their avatar, and has alt text.

**Comment by Jbarget** [comment]
![screen shot 2017-09-15 at 12 08 14](https://user-images.githubusercontent.com/11725595/30479883-062c0220-9a0f-11e7-8e88-2bebab267851.png)
im not getting the assignee image :( 

Here's the markup:
![screen shot 2017-09-15 at 12 12 04](https://user-images.githubusercontent.com/11725595/30479909-2273b1bc-9a0f-11e7-97d6-3199d658a2cf.png)

**Comment by Jbarget** [comment]
is this old behaviour, i've pulled down the latest `master`

**Comment by Jbarget** [comment]
ah realised i probably need to purge my DB
